independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/strategy/ReplaceCodestartFileStrategyHandler.java

```diff
@@ -28,7 +28,6 @@ public void process(Path targetDirectory, String relativePath, List<TargetFile>
             throws IOException {
         checkNotEmptyCodestartFiles(codestartFiles);
         final Path targetPath = targetDirectory.resolve(relativePath);
-        checkTargetDoesNotExist(targetPath);
```

Comment:
Why?

Reply:
Because, this throws an exception preventing the actual file replacement.

Reply:
This shouldn't be the case as the file shouldn't exist before this is called.

Reply:
... and throwing an exception if file exists doesn't sound like replace to me. 

What do I miss?

Reply:
Look just bellow the replace is working by using the latest file in the List